src/org/parosproxy/paros/network/SSLConnector.java

```diff
@@ -374,8 +375,19 @@ public Socket createSocket(final String host, final int port,
 			}
 			try {
 				SSLSocket sslSocket = (SSLSocket) clientSSLSockFactory.createSocket(host, port, localAddress, localPort);
-				sslSocket.startHandshake();
-
+				try {
+					sslSocket.startHandshake();
+				} catch (SSLException ex) {
+					String errHandshakeFailHelpMsg = Constant.messages.getString("network.sslconnector.handshake.error.help",
+							Constant.messages.getString("network.sslconnector.handshake.error.help.url"));
+					logger.warn("An exception occurred while attempting to connect to: " + sslSocket.getInetAddress());

```

Comment:
Does it worth logging if we are able to connect in the end? (per following catch logic)